### ISSUE TITLE
Unable to access transferred files (corporate computer requires admin account to install)

### ISSUE THREAD

**Issue by idoric** [swiszczoo/winpinator#12]
Steps to Reproduce:
- log on to the computer as a normal user
- install on a computer A that requires an administrator username and password to be able to install anything
- transfer files and directories from a computer B to computer A via Winpinator
- try to "open" or "show in explorer…" these files and directories

Actual result:
For a file, nothing happens. For a directory, nothing happens with "show in explorer…", but with "open" Windows says that we aren't authorized to access the directory C:\Users\<admin account>\Documents\Winpinator\<loaded directory>.

Expected result:
Be able to access and open files and directories that have been transferred to the computer, for example in C:\Users\<normal user account>\Documents\Winpinator\<loaded directory>.

Long story short, my administrator kindly entered his password for the installation, but he won't give access to the directory which belongs to the admin account.

Proposed solution: create a Winpinator directory per user, in his personal directory, in order to have no problem with reading and writing rights.

**Comment by swiszczoo** [comment]
Hi, I haven't tried to replicate it yet but if it works like that, it's definitely a bug (it's supposed to use your own Documents directory). Can you check if this workaround works for you? Go to `Preferences` and under `General` tab there is a box called `Location for received files`, where you can select change your save path to some dir that belongs to your account. It should be stored for each Windows user separately.
Thanks.

**Comment by idoric** [comment]
I will test the workaround as soon as possible, which means not before two weeks unfortunately.

**Comment by idoric** [comment]
Sorry for the wait, I have now been able to test the workaround, and I confirm that it works.
